### PR TITLE
Made usecases more explicit from user feedback

### PR DESCRIPTION
**Pull request by checklyalex**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

### CODE REVIEW

File: site/content/docs/private-locations/_index.md [hunk 1]
```diff
@@ -11,7 +11,12 @@ aliases:
 cli: true
 ---
 
-A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure. A Private Location can be used to run checks on your services which are not directly exposed to the public internet but in your own segregated network. To operate a Private Location you need to run at least one Checkly Agent.
+A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure. To operate a Private Location you need to run at least one Checkly Agent.
+
+Private Locations allow you to run checks from within your own infrastructure, enabling testing and monitoring of both internal services (within your private network) and external services (publicly accessible).
+
+1. Internal Monitoring: Test the availability and performance of services that aren't accessible externally.
```
Comment (ragog) [inline site/content/docs/private-locations/_index.md:18]: @checklyalex this numbered list's connection to the previous paragraph is not obvious. Do we need a colon someplace?

File: site/content/docs/private-locations/_index.md [hunk 2]
```diff
@@ -11,7 +11,12 @@ aliases:
...
+2. External Monitoring: Simulate user experience from specific geographic locations of your choosing
```
Comment (ragog) [inline site/content/docs/private-locations/_index.md:19]: @checklyalex period missing at the end of phrase.

**Review by ragog — CHANGES_REQUESTED** [review]
@checklyalex let's rewrite this more precisely.

File: site/content/docs/private-locations/_index.md [hunk 3]
```diff
@@ -11,12 +11,18 @@ aliases:
 cli: true
 ---
 
-A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure. A Private Location can be used to run checks on your services which are not directly exposed to the public internet but in your own segregated network. To operate a Private Location you need to run at least one Checkly Agent.
+A Private Location is a monitoring location that you set up and manage within your own network. The key difference is that you install a lightweight Checkly Agent on your own server or infrastructure.
```
Comment (ragog) [inline site/content/docs/private-locations/_index.md:14]: @checklyalex necessarily "within your internal network"? Later on we mention these can be spun up from different parts of the world, for example...

**Review by ragog — CHANGES_REQUESTED** [review]
@checklyalex one comment

File: site/content/docs/private-locations/_index.md (hunk 3 above)
Comment (checklyalex) [reply site/content/docs/private-locations/_index.md:14]: Ok I will change to "A Private Location is a monitoring location that you manage by simply deploying a lightweight Checkly Agent."

**Review by ragog — APPROVED** [review]
@checklyalex LGTM

**Pull request merged**